File: backend/model/Estadisticas.py
import json

class Estadisticas_Descriptiva(object):
    moda = 0
    media = 0
    mediana = 0
    desv_estandar = 0
    varianza = 0

class Estadisticas(object):
    estadisticas_t_arribo: Estadisticas_Descriptiva
    estadisticas_t_servicio: Estadisticas_Descriptiva

    def __init__(self):
        self.estadisticas_t_arribo = Estadisticas_Descriptiva()
        self.estadisticas_t_servicio = Estadisticas_Descriptiva()

    def toJSON(self):
        return json.dumps(self, default = lambda o:o.__dict__, 
            sort_keys = True, indent = 4)

File: frontend/src/app/Model/Estadisticas.ts
export class Estadisticas {
    estadisticas_t_arribo: EstadisticasDescriptivas;
    estadisticas_t_servicio: EstadisticasDescriptivas;
    histograma_t_arribo: DatoHistograma[];
    histograma_t_servicio: DatoHistograma[];
}

export class DatoHistograma {
    dato: number;
    index: number;
    rango: string;
}

export class EstadisticasDescriptivas {
    desv_estandar: number;
    media: number;
    mediana: number;
    moda: number;
    varianza: number;
}
